src/System.IO.Compression/src/System/IO/Compression/GZipStream.cs

```diff
@@ -10,42 +10,26 @@ namespace System.IO.Compression
     public class GZipStream : Stream
     {
         private DeflateStream _deflateStream;
+        private const int _gzipWindowBits = 31;
```

Comment:
Good call. I'm going to move this to ZLibNative and add some detailed doc comments like we have for Deflate_DefaultWindowBits